sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -39,10 +40,23 @@ namespace Azure { namespace Core {
      */
     std::string ToString();
 
+    /**
+     * @brief Returns the internal value of the Uuid for consumption by clients who need non-string
```

Comment:
Nit: `internal` is a loaded term within the SDK and that is an implementation detail which doesn't need to be exposed. We _could_ change the "internal" representation of Uuid in the future, but still retain this function signature as is (with a data copy/etc.).
```suggestion
     * @brief Returns the binary value of the Uuid for consumption by clients who need non-string
```